code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -66,7 +74,7 @@
 /obj/item/weapon/extinguisher/proc/AttemptRefill(atom/target, mob/user)
 	if(istype(target, /obj/structure/reagent_dispensers/watertank) && target.Adjacent(user))
 		var/safety_save = safety
-		safety = 1
+		safety = TRUE

```

Comment:
I personally think you should have to turn the safety on yourself before refilling it. What if the water tank is on fire?